colossalai/trainer/hooks/_checkpoint_hook.py

```diff
@@ -17,9 +14,8 @@ class SaveCheckpointHook(BaseHook):
     """Saves the model by interval in training process.
 
     Args:
-       interval (int, optional): Saving interval, defaults to 1.
-       checkpoint_dir (str, optional): Directory of saving checkpoint, defaults to None.
-       suffix (str, optional): Saving suffix of the file, defaults to ''.
+       interval (int, optional): Number of epochs between saving the checkpoint, defaults to 1.
```

Comment:
why in saving in the interval of epoch?
What about using iterations.

Reply:
We are absolutely able to do so. Just add an `after_train_step` function for this hook.